src/Mod/TechDraw/App/ProjectionAlgos.h

```diff
@@ -0,0 +1,86 @@
+/***************************************************************************
+ *   Copyright (c) Jürgen Riegel          (juergen.riegel@web.de) 2009     *

```

Comment:
Fix heading formatting (should look like [this](https://github.com/FreeCAD/FreeCAD/pull/5457/files#diff-06fc5002222d6a7ea53e9ddb9a565cf709e7ea9ec62841246412d436f05a6edbR2))